content/cli/workspaces.md

```diff
@@ -190,6 +191,76 @@ These properties specify the compiler to use as well as various options that aff
 | `deleteOutDir`      | boolean             | If `true`, whenever the compiler is invoked, it will first remove the compilation output directory (as configured in `tsconfig.json`, where the default is `./dist`).                                                                    |
 | `assets`            | array               | Enables automatically distributing non-TypeScript assets whenever a compilation step begins (asset distribution does **not** happen on incremental compiles in `--watch` mode).  See below for details.                                  |
 
+#### Global generate options
+
+These properties specify the default generate options for when running the `nest generate` command.
+
+| Property Name       | Property Value Type | Description                                                                                                                                                                                                                              |
+| ------------------- | ------------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `spec`              | boolean *or* object | If assigned to a boolean it can be specified as either `true` or `false` to respectively enable or disable `spec` generation by default, if not specified otherwise in the cli, or in the specific project `generateOptions`. If assigned to an object, each key represents a schematic name, and the boolean value represents if the default spec generation is enabled / disabled for that specific schematic. |
+
+Below is given an example of global generate options, both with spec being a boolean and as an object
+
+```javascript
+{
+  "generateOptions": {
+    "spec": false
+  },
+  ...
+}
+```
+
+or 
+
+```javascript
+{
+  "generateOptions": {
+    "spec": {
+      "service": false
+    }
+  },
+  ...
+}
+```
+
+> error **Warning** Specifying the `spec` as an object, the key for the generation schematic does currently not support automatic alias handling. This means that specifying a key as for example `service: false` and trying to generate a service via the alias `s`, the spec would still be generated. To make sure both the normal schematic name and the alias works, specify both the normal command name as well as the alias, as seen below.
```

Comment:
"Specifying the `spec` " -> "When Specifying the `spec` "

"does currently not" -> "does not currently"

"the alias works" -> "the alias work as intended'